src/EquivalenceCheckingManager.cpp

```diff
@@ -196,6 +196,16 @@ EquivalenceCheckingManager::EquivalenceCheckingManager(
     fixOutputPermutationMismatch();
   }
 
+  // check whether the alternating checker is configured and can handle the
+  // circuits
+  if (configuration.execution.runAlternatingChecker &&
+      !DDAlternatingChecker::canHandle(this->qc1, this->qc2)) {
+    std::clog << "[QCEC] Warning: alternating checker cannot handle the "
```

Comment:
This is a more general question regarding configurations. If a user has disabled the construction checker this will overwrite that configuration. That is the same as it is handled right now when checking parameterized circuits (ZX-checker will always be used, even if disabled). Should we maybe make a distinction between default settings and user specified settings? So if a user requests the construction checker to be off (for whatever reason) should we keep it turned off? 

Not really something to be handled by this PR, but I wanted to note that somewhere.

Reply:
I added a reference to this conversation to #146 and will resolve this conversation.